src/frontend/screens/ObservationsList/ProjectCard.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {BLUE_GREY, VERY_LIGHT_GREY} from '../../lib/styles';
+import {useProjectSettings} from '@comapeo/core-react';
+import {useActiveProject} from '../../contexts/ActiveProjectContext';
+import {defineMessages, useIntl} from 'react-intl';
+import {useOwnRoleInProject} from '@comapeo/core-react';
+import {
+  COORDINATOR_ROLE_ID,
+  CREATOR_ROLE_ID,
+  MEMBER_ROLE_ID,
+} from '../../sharedTypes';
+import ProjectCoordinatorIcon from '../../images/ProjectCoordinator.svg';
+import ProjectParticipantIcon from '../../images/ProjectParticipant.svg';
+import NoProjectIcon from '../../images/NoProjectIcon.svg';
+import {BodyText} from '../../sharedComponents/Text/BodyText';
+
+const m = defineMessages({
+  mappingOnYourOwn: {
+    id: 'screens.ObservationsList.ProjectCard.mappingOnYourOwn',
+    defaultMessage: 'You’re mapping on your own.',
+  },
+  coordinator: {
+    id: 'screens.ObservationsList.ProjectCard.coordinator',
+    defaultMessage: 'You’re a coordinator on this project.',
+  },
+  participant: {
+    id: 'screens.ObservationsList.ProjectCard.participant',
+    defaultMessage: 'You’re a participant on this project.',
+  },
+});
+
+export function ProjectCard() {
+  const {projectId} = useActiveProject();
+  const {data} = useProjectSettings({projectId: projectId});
+
+  const {formatMessage} = useIntl();
+  const {data: role} = useOwnRoleInProject({
+    projectId: projectId,
+  });
+
+  let projectRole = 'solo';
+  if (data.name) {
+    if (
+      role.roleId === COORDINATOR_ROLE_ID ||
+      role.roleId === CREATOR_ROLE_ID
+    ) {
+      projectRole = 'coordinator';
+    } else if (role.roleId === MEMBER_ROLE_ID) {
+      projectRole = 'participant';
+    }
+  }
+
+  let userRoleText, RoleIcon;
+  switch (projectRole) {
+    case 'coordinator':
+      userRoleText = formatMessage(m.coordinator);
+      RoleIcon = ProjectCoordinatorIcon;
+      break;
+    case 'participant':
+      userRoleText = formatMessage(m.participant);
+      RoleIcon = ProjectParticipantIcon;
+      break;
+    default:
+      userRoleText = formatMessage(m.mappingOnYourOwn);
+      RoleIcon = NoProjectIcon;
+      break;
+  }
+
+  return (
+    <View style={styles.card}>
+      <RoleIcon width={20} height={20} />
+      <BodyText testID="OBS.card-text" variant="smallMeta">
+        {userRoleText}
+      </BodyText>
```

Comment:
the text is overflowing a bit (just ever so slightly).:
<img width="293" alt="image" src="https://github.com/user-attachments/assets/e8956e72-7b7e-4a4d-8eac-99f47f692a00" />